doc/introduction/chemistry.rst

```diff
@@ -0,0 +1,191 @@
+Quantum Chemistry
+=================
+
+PennyLane provides a quantum chemistry module ``qchem`` to perform quantum simulations
+of the electronic structure of molecules. ``qchem`` contains tools to construct the
+electronic Hamiltonian of molecules, and uses PennyLane to implement the Variational Quantum
+Eigensolver (VQE) algorithm.
+
+
+.. note::
+
+    To access the ``qchem`` module, the ``PennyLane-QChem`` plugin must be installed separately:
+
+    .. code-block::
+
+        pip install pennylane-qchem
+
+Building the electronic Hamiltonian
+-----------------------------------
+
+The ``qchem`` module provides access to a driver function :func:`~.generate_mol_hamiltonian`
+to generate the electronic Hamiltonian in a single call. For example,
+
+.. code-block:: python
+
+    h, nr_qubits = qml.qchem.generate_mol_hamiltonian(
+        name='h2',
+        geo_file='h2.xyz',
+        charge=0,
+        multiplicity=1,
+        basis_set='sto-3g',
+        n_active_electrons=2,
+        n_active_orbitals=2
+    )
+
+where:
+
+* ``h`` is the qubit Hamiltonian of the molecule represented as an
+  `OpenFermion <https://github.com/quantumlib/OpenFermion>`_ ``QubitOperator``, and
+
+* ``nr_qubits`` is the number of qubits operators needed to represent it.
+
+Internally, :func:`~.generate_mol_hamiltonian` calls the following functions in order
+to generate the qubit Hamiltonian:
+
+.. currentmodule:: pennylane_qchem.qchem
+
+.. autosummary::
+
+    read_structure
+    gen_meanfield_data
+    gen_active_space
+    gen_hamiltonian_pauli_basis
+
+
+For more fine-grained control, these functions may be
+called independently as required to obtain the

```

Comment:
I would maybe end the sentence at "... called independently as required."

Reply:
```suggestion
called independently as required.
```